.github/workflows/python_test.yml

```diff
@@ -0,0 +1,38 @@
+name: Python application test
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip3 install -e .
+        sudo curl -sSL http://get.gazebosim.org | sh
+        sudo apt install python3-tk xvfb
+        git submodule update --init --recursive
+        mkdir build
+        cd build
+        cmake ..
+        make -j
+        cd ..
+        pip3 install -e .
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 ./python --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
It's better to follow https://github.com/HorizonRobotics/alf/blob/master/.ci-cd/build.sh to use the same style check to be consistent with ALF style.

Reply:
Added the the same style check using pre-commit. 
This flake8 checking can be kept, could do static checking for syntax errors or undefined names in corner case that tests does not cover, also it can report warnings like unused import, variables assigned to but never used. max-line-length checking can be ignored since pre-commit already checked that.